packages/website/docs/Features/Language.mdx

```diff
@@ -1,39 +1,51 @@
 ---
 sidebar_position: 2
+title: Language Basics
 ---
 
 import { SquiggleEditor } from "../../src/components/SquiggleEditor";
 
-# Squiggle Language
-
 The squiggle language has a very simple syntax. The best way to get to understand
 it is by simply looking at examples.
 
-## Basic Language
+## Expressions and statements
+
+A squiggle **expression** is a value like a float or a distribution or a data structure like an array or a record.
+
+<SquiggleEditor initialSquiggleString={`mixture(1 to 2, 3, [0.3, 0.7])`} />
 
-As an example:
+The bottom line of your squiggle buffer should be an expression, which we evaluate (i.e., render). Sometimes we call the last expression of a squiggle file an _export_.
+
+A squiggle **assignment** is a statement, when you want to bind an expression to a name.
```

Comment:
I feel like assignments are pretty obvious. Discussing "bind" is getting into the weeds a bit.